integration/networking/port_mapping_linux_test.go

```diff
@@ -251,10 +252,22 @@ func TestProxy4To6(t *testing.T) {
 	)
 	defer c.ContainerRemove(ctx, serverId, containertypes.RemoveOptions{Force: true})
 
+	poll.WaitOn(t, container.IsInState(ctx, c, serverId, "running"),
+		poll.WithDelay(100*time.Millisecond))
+
 	inspect := container.Inspect(ctx, t, c, serverId)
 	hostPort := inspect.NetworkSettings.Ports["80/tcp"][0].HostPort
 
 	resp, err := http.Get("http://[::1]:" + hostPort)
+	var err error
+	var resp *http.Response
+	for retry := 0; retry < 10; retry++ {
+		resp, err = http.Get("http://[::1]:" + hostPort)
+		if err == nil {
+			break
+		}
+		time.Sleep(100 * time.Millisecond)
```

Comment:
I'm unsure if this was useful too. I think maybe it is, but I can't remember now. We should test it, probably.